kadi/commands/command_sets.py

```diff
@@ -0,0 +1,263 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import re
+from pathlib import Path
+
+import astropy.units as u
+
+from Quaternion import Quat
+from cxotime import CxoTime
+from parse_cm.common import _coerce_type as coerce_type
+from kadi.commands.core import CommandTable
+
+RTS_PATH = Path('FOT/configuration/products/rts')
+
+
+def cmd_set_rts(*args, date=None):
+    from parse_cm.csd import csd_cmd_gen
+
+    rts = 'SCS_CATEGORY,OBSERVING\n' + '\n'.join(args).upper()
+    cmds = csd_cmd_gen(rts, date=date)
+    return cmds
+
+
+def cmd_set_obsid(obs_id, date=None):
```

Comment:
And are a bunch of these set to take dates they don't use to make the API simpler?  Would generic kwargs make sense at that point?

Reply:
Yeah, this isn't the most beautiful. But adding a generic `kwargs` doesn't really help from what I can see. You still get the linter or reader of the code complaining that `kwargs` is not being used. As it stands it is clear that every `cmd_set_*` function must accept an optional `date` and nothing else. With `kwargs` then the function of that unused keyword is less clear.